DockerfileSlave

```diff
@@ -7,10 +7,11 @@ USER root
 RUN apt-get update \
     && apt-get install -y apt-transport-https ca-certificates \
     && apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D \
-    && echo deb https://apt.dockerproject.org/repo debian-jessie main >> /etc/apt/sources.list.d/docker.list \
+    && echo deb https://apt.dockerproject.org/repo debian-stretch main >> /etc/apt/sources.list.d/docker.list \
     && apt-get update \
     && apt-get install -y docker-engine \
     && apt-get install -y btrfs-tools \
+    && apt-get install openjdk-11-jdk-headless \
```

Comment:
OpenJDK should already be installed by the main image, no ?

Reply:
Only the JRE